### PR TITLE
update presubmit tests sh file

### PR DESCRIPTION
**Pull request by beemarie**

### CODE REVIEW

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+
+export GO111MODULE=on
+
+# Markdown linting failures don't show up properly in Gubernator resulting
+# in a net-negative contributor experience.
+export DISABLE_MD_LINTING=1
+
+source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/presubmit-tests.sh
+
+# TODO(mattmoor): integration tests
```
Comment (tcnghia) [inline test/presubmit-tests.sh:32]: Create an issue and refer here?

Comment (beemarie) [reply test/presubmit-tests.sh]: @tcnghia I noticed most of the other presubmit-test.sh files in other knative-sandbox repos had this todo here, and was thinking maybe it was left for a reason. We can just remove.

File: test/presubmit-tests.sh [hunk 2]
```diff
@@ -1,4 +1,33 @@
 #!/usr/bin/env bash
 
-# TODO(approvers): replace with real tests
-true
+# Copyright 2019 The Knative Authors
```
Comment (maximilien) [inline test/presubmit-tests.sh:3]: Update Copyright year to 2020

**Review by maximilien — CHANGES_REQUESTED** [review]
Small change. Otherwise LGTM.

**Review by maximilien — COMMENTED** [review]
/lgtm

**Pull request merged**